src/main/resources/config.yml

```diff
@@ -106,5 +106,3 @@ items:
     next_button: ''
     cancel_button: "DEFAULT"
     compass_button: "DEFAULT"
-profiles:
-  default_name: 'default'
```

Comment:
Why do we now have no default name anymore? or do we readd it, when the commands are comming?